EPR/Library/tests/Makeray.txt

```diff
@@ -3,6 +3,9 @@ CFLAGS = -Wall -g
 
 test_rays: 
 	$(CC) $(CFLAGS) test_rays.cpp  doctest.h ../rays.cpp ../rays.hpp -o test_rays
+test_noise: 
+	$(CC) $(CFLAGS) test_noise.cpp  doctest.h ../noise.cpp ../noise.hpp -o test_noise
 
 clean:
-	rm -f test_rays
\ No newline at end of file
+	rm -f test_rays
```

Comment:
I personally like to have all of the files that will be removed on one line, but I can see it the other way, too.  Should we be consistent?  Also, should we integrate into one makefile for the tests directory, or should we keep it split like this?

Reply:
I think we should integrate into one makefile, but I am not that experienced with makefiles and could not get them to work inside 'Makefile' so that why I created 'Makeray'. If someone can integrate that on their next push, it would be awesome.